.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true
+}
```

Comment:
maybe delete (?) Now that it's commited, I don't know how to "remove" it if not by deleting it. Hmm.